### PR TITLE
feature - init tabbar controller

### PR DESCRIPTION
**Pull request by at-sonvu**

### CODE REVIEW

**Comment by at-daiho** [comment]
:+1: There are no linter violations.

File: MyApp/View/Controllers/Tabbar/TabbarViewController.swift [hunk 1]
```diff
@@ -7,10 +7,41 @@
...
+        tabPersonal.tabBarItem = tabPersonalBarItem
+        let tabPersonalNavigation = UINavigationController(rootViewController: tabPersonal)
+
+        // tabSetting
+        let tabSetting = SettingViewController()
+        let tabSettingBarItem = UITabBarItem(title: App.String.setting, image: #imageLiteral(resourceName: "setting"), selectedImage: #imageLiteral(resourceName: "setting"))
+        tabSetting.tabBarItem = tabSettingBarItem
+        let tabSettingNavigation = UINavigationController(rootViewController: tabSetting)
+
+        viewControllers = [tabHomeNavigation, tabSearchNavigation, tabPersonalNavigation, tabSettingNavigation]
```
Comment (at-tuanmai2) [inline MyApp/View/Controllers/Tabbar/TabbarViewController.swift]: please move to new func and call in viewDidLoad

File: MyApp/View/Controllers/Tabbar/TabbarViewController.swift [hunk 2]
```diff
@@ -7,10 +7,41 @@
...
 
     override func viewDidLoad() {
         super.viewDidLoad()
+        UITabBar.appearance().tintColor = UIColor.purple
+    }
+
+    override func viewWillAppear(_ animated: Bool) {
+        super.viewWillAppear(animated)
+        // tabHome
+        let tabHome = HomeViewController()
```
Comment (at-tuanmai2) [inline MyApp/View/Controllers/Tabbar/TabbarViewController.swift]: change name

File: MyApp/View/Controllers/Tabbar/TabbarViewController.swift [hunk 3]
```diff
@@ -7,10 +7,41 @@
...
+
+        let tabHomeBarItem = UITabBarItem(title: App.String.home, image: #imageLiteral(resourceName: "home"), selectedImage: #imageLiteral(resourceName: "home"))
+        tabHome.tabBarItem = tabHomeBarItem
+        let tabHomeNavigation = UINavigationController(rootViewController: tabHome)
+
+        // tabSearch
+        let tabSearh = SearchViewController()
```
Comment (at-tuanmai2) [inline MyApp/View/Controllers/Tabbar/TabbarViewController.swift]: change name

File: MyApp/View/Controllers/Tabbar/TabbarViewController.swift [hunk 4]
```diff
@@ -7,10 +7,44 @@
...
+class TabbarViewController: UITabBarController {
 
     override func viewDidLoad() {
         super.viewDidLoad()
+        UITabBar.appearance().tintColor = UIColor.purple
+    }
+
+    override func viewWillAppear(_ animated: Bool) {
+        super.viewWillAppear(animated)
+        initTabbarVC()
```
Comment (at-tuanmai2) [inline MyApp/View/Controllers/Tabbar/TabbarViewController.swift]: call in viewDidLoad, remove func viewWillAppear if no need

**Pull request merged**